lib/src/services/sembast.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+import 'package:sembast/sembast_io.dart';
+import 'package:path_provider/path_provider.dart';
+
+late Database _database;
+
+Future<void> setupSembast() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  final appDir = await getApplicationDocumentsDirectory();
+  final dbPath = '${appDir.path}/app_database.db';
+  _database = await databaseFactoryIo.openDatabase(dbPath);
+}
```

Comment:
We do support web in the template, but `sembast` doesn't support it. Please, include [`sembast_web`](https://pub.dev/packages/sembast_web) and provide setup logic to make sure the example app does work on both mobile and web.